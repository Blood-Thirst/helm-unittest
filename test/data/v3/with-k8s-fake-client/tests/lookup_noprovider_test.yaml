templates:
  - templates/lookup.yaml
tests:
  - it: manifest should match snapshot
    asserts:
      - matchSnapshot: {}
      - isNullOrEmpty:
          path: pod_exists
      - isNullOrEmpty:
          path: ingress_exists
      - isNullOrEmpty:
          path: namespace_exists
      - isNullOrEmpty:
          path: pod_not_exists
      - isNullOrEmpty:
          path: ingress_not_exists
      - isNullOrEmpty:
          path: namespace_not_exists
