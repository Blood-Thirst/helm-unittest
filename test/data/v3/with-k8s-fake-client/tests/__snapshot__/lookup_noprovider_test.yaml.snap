manifest should match snapshot:
  1: |
    ingress_exists: {}
    ingress_not_exists: {}
    namespace_exists: {}
    namespace_not_exists: {}
    namespaces: {}
    pod_exists: {}
    pod_not_exists: {}
